### PR TITLE
Add isEmpty / isNotEmpty for collections methods

### PR DESCRIPTION
**Pull request by leleuj**

### CODE REVIEW

File: pac4j-core/src/main/java/org/pac4j/core/util/CommonHelper.java [hunk 1]
```diff
@@ -86,6 +87,26 @@ public static boolean areNotEquals(final String s1, final String s2) {
...
+    }
+
+    /**
+     * Return if a collection is not empty.
+     *
+     * @param coll a collection
+     * @return whether it is not empty
+     */
+    public static boolean isNotEmpty(final Collection coll) {
+        return coll != null && !coll.isEmpty();
```
Comment (mmoayyed) [inline pac4j-core/src/main/java/org/pac4j/core/util/CommonHelper.java]: You could simply turn this into !isEmpty()

Comment (leleuj) [inline pac4j-core/src/main/java/org/pac4j/core/util/CommonHelper.java]: I want to be able to check that without worrying about the fact that the collection can be null. Hence the first check.

Comment (mmoayyed) [inline pac4j-core/src/main/java/org/pac4j/core/util/CommonHelper.java]: To clarify, my suggestion was not so much:

```
return !coll.isEmpty()
```

But more like:

```
return !isEmpty(coll)
```

I believe this is what commons does too.

Comment (leleuj) [inline pac4j-core/src/main/java/org/pac4j/core/util/CommonHelper.java]: OK. I get it now. Will change.

**Pull request merged**